pkg/agent/flowexporter/connections/deny_connections.go

```diff
@@ -21,6 +21,7 @@ import (
 	"k8s.io/klog"
 
 	"github.com/vmware-tanzu/antrea/pkg/agent/flowexporter"
+	"github.com/vmware-tanzu/antrea/pkg/util/ip"
 )
 
 type DenyConnectionStore interface {
```

Comment:
Whats the reason for creating this interface? Are there any mocks used somewhere for testing?